### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by svrana**

### CODE REVIEW

File: broker.go [hunk 1]
```diff
@@ -193,16 +212,23 @@ type OnQueueCreation func(qm *QueueManager, q Queue, name QueueName)
...
 	// If specified the defaultConsumer is used when a consumer has not
 	// been specified in the queuedef.
 	defaultConsumer MessageConsumer
+
+	metricNamespace     string
+	msgInflight         map[Queue]*prometheus.GaugeVec
+	msgOutstanding      map[Queue]*prometheus.GaugeVec
+	msgSendFailures     map[Queue]*prometheus.CounterVec
+	msgSendSuccesses    map[Queue]*prometheus.CounterVec
+	msgConsumeSuccesses map[Queue]*prometheus.CounterVec
```
Comment (svrana) [inline broker.go:229]: Just need one Vector here, not one for each queue...

**Comment by rkotcher-synthesisai** [comment]
@svrana did you mean to squash 466bfe97103725a5170a6d499611a6279fc6edf6?

**Review by rkotcher-synthesisai — APPROVED** [review]
maybe just squash that one commit, but otherwise looks great

**Comment by svrana** [comment]
> @svrana did you mean to squash [466bfe9](https://github.com/Synthesis-AI-Dev/broker/commit/466bfe97103725a5170a6d499611a6279fc6edf6)?

No, but didn't mean to include it here. It's just removing a lint warning unrelated to metrics.

**Pull request merged**